src/installer/pkg/packaging/osx/package.targets

```diff
@@ -13,6 +13,11 @@
   <Target Name="GeneratePkgs"
           Condition="'$(OSGroup)' == 'OSX'"
           DependsOnTargets="GetInstallerBrandingNames;InitPkg">
+    <PropertyGroup>
+        <SharedFrameworkInstallerFile Condition="!$(SharedFrameworkInstallerFile.EndsWith('.pkg'))">$(SharedFrameworkInstallerFile).pkg</SharedFrameworkInstallerFile>
+        <SharedHostInstallerFile Condition="!$(SharedHostInstallerFile.EndsWith('.pkg'))">$(SharedHostInstallerFile).pkg</SharedHostInstallerFile>
+        <HostFxrInstallerFile Condition="!$(HostFxrInstallerFile.EndsWith('.pkg'))">$(HostFxrInstallerFile).pkg</HostFxrInstallerFile>
```

Comment:
Will fix - it seems VSCode messed this one.

Reply:
I have this in my settings to prevent that. 🙂 

```
    "files.associations": {
        "*.msbuild": "xml",
        "*.depproj": "xml",
        "*.pkgproj": "xml",
        "*.sfxproj": "xml",
        "*.bundleproj": "xml",
        "*.builds": "xml",
        "*.proj": "xml",
        "*.props": "xml",
        "*.targets": "xml",
...
    },
```

```
    "[xml]": {
        "editor.tabSize": 2
    },
```

Reply:
Indents match the rest of this file, will keep this as is.